nixos/modules/services/desktops/flatpak.nix

```diff
@@ -20,12 +20,7 @@ in {
         type = types.listOf types.package;
         default = [];
         description = ''
-          List of additional portals to add to path. Portals allow interaction
-          with system, like choosing files or taking screenshots. At minimum,
-          a desktop portal implementation should be listed. GNOME already
-          adds <package>xdg-desktop-portal-gtk</package>; for KDE, there
-          is <package>xdg-desktop-portal-kde</package>. Other desktop
-          environments will probably want to do the same.
+          DEPRECATED,please use <literal>xdg.portal.extraPortals</literal>
```

Comment:
Perhaps just delete it and use [mkRenamedOptionModule](https://github.com/NixOS/nixpkgs/blob/2143f6f34fb715e8d987624fd1d5b087edb1e487/nixos/modules/rename.nix)?